### DISCUSSION TITLE
Is it possible to use unocss icons without unocss?

### DISCUSSION THREAD

**Discussion by AxelBriche** [unocss/unocss#1892] [Q&A]
Hi, I create my project with Remix and use vanilla css.
I would like to know if it is possible to use @unocss/preset-icons without unocss preset like windy?

**Comment by userquin** [comment]
yes: you only need unocss plugin (vite,webpack...) and @unocss/preset-icons, you don't need any other preset

**Reply by AxelBriche** [reply]
I don't use Webpack or Vite, but I use Postcss, I don't know if they have a plugin for that or for use unocss without "asset manager/builder" only with npm and command line.

**Reply by userquin** [reply]
you need a `bundler` or use the `cli`: https://github.com/unocss/unocss/tree/main/packages/cli
